docker/backup.postgres.plugin

```diff
@@ -5,6 +5,24 @@
 # -----------------------------------------------------------------------------------------------------------------
 export serverDataDirectory="/var/lib/pgsql/data"
 
+# Override general function to support defining a schema
+function getDatabaseName(){
+  (
+    _databaseSpec=${1}
+    _databaseName=$(echo ${_databaseSpec} | sed -n 's~^.*/\([^?]*\).*~\1~p')
+    echo "${_databaseName}"
+  )
+}
+
+function getDatabaseSchema(){
+  (
+    _databaseSpec=${1}
+    _paramName=$(echo ${_databaseSpec} | sed -n  's~^.*currentSchema=\(.*$\)~\1~p')
+    _databaseSchema="${!_paramName:-public}"
+    echo ${_databaseSchema}
+  )
+}
+
```

Comment:
I'd feel more comfortable with these going in `backup.config.utils` as the standard implementation so these features are available to all implementations.

I would also recommend changing `currentSchema` to just `schema`.

Reply:
Will refactor it out. I opted to put it here as both MongoDB and MariaDB do not support the concept of schema - it is "flattened" with the concept of database in both cases, as far as I know.